packages/features/form-builder/fieldTypes.ts

```diff
@@ -104,6 +104,9 @@ const configMap: Record<FieldType, Omit<z.infer<typeof fieldTypeConfigSchema>, "
     label: "Long Text",
     value: "textarea",
     isTextType: true,
+    supportsLengthCheck: {
+      maxLength: 4000,
+    },
```

Comment:
@mhetreayush Added maxLength support here. It easily allows any field that wants to support length check feature to define maxLength. Infact it is mandatory to define maxLength if someone enables length check support.

This should also solve the problem of easily configuring it for **text** field type.

Reply:
This is different from the `maxLength` that we have in `baseFieldSchema` as that can be changed by the user per field. This on the other hand is constant and defined by our code and applies to all fields of that type